templates/bake/element/Controller/delete.twig

```diff
@@ -17,7 +17,7 @@
      * Delete method
      *
      * @param string|null $id {{ singularHumanName }} id.
-     * @return \Cake\Http\Response|null|void Redirects to index.
+     * @return \Cake\Http\Response|null Redirects to index.
      * @throws \Cake\Datasource\Exception\RecordNotFoundException When record not found.
      */
     public function delete($id = null)
```

Comment:
We could probably generate the actual return type since `mixed|void` can be narrowed to `?Response`.

But, this works in general.